src/Dynode/model/core/node.ts

```diff
@@ -115,7 +127,7 @@ class Node extends VEventTarget {
    *
    * @param name  The name of the input of which the value will be retrieved.
    */
-  getInputValue(name: string): unknown {
+  getInputValue(name: string): Value<unknown> {
     const input = this.getInput(name);
     return input.getValue();
```

Comment:
Since Value class is going to be renamed, socket's getValue should also reflect the same nomenclature. I would say that getInputValue should in fact return raw value, though input.getValue() should do the same (otherwise it should be named differently)